io.openems.edge.goodwe.et/src/io/openems/edge/goodwe/et/ess/Config.java

```diff
@@ -27,6 +27,9 @@
 
 	@AttributeDefinition(name = "Modbus-ID", description = "ID of Modbus brige.")
 	String modbus_id() default "modbus0";
+	
+	@AttributeDefinition(name = "Ess-Capacity", description = "capacity of the battery in Wh")
```

Comment:
Call it "Capacity", we are already within an "Ess". Start description with capital letter.

Reply:
Ok, I will correct it. 
I purposefully set it because there is a possibility to enter inverter capacity whoever is initialising it. If a person is referencing the Sems portal for values, then it is mentioned as just capacity in the portal, and it refers to Inverter capacity, not the battery.